pkgs/development/python-modules/solo-python/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchFromGitHub
+, click, ecdsa, fido2, intelhex, pyserial, pyusb, requests}:
+
+ buildPythonPackage rec {
+  pname = "solo-python";
+  version = "0.0.15";
+  format = "flit";
+
```

put disable logic here :). This will allow 3.6, and 3.7, but not <=3.5
```suggestion
    disabled = pythonOlder "3.6";

```

---

Done